### DISCUSSION TITLE
Q: project contribution

### DISCUSSION THREAD

**Discussion by hliberacki** [heremaps/gluecodium#1601] [General]
I thought about making contribution to the project, but I can see that the youngest issue is 11mo. Are they still aligned with the needs/roadmap?

If there is any issue that is more pressing to be implemented then I could try to do so, possible suggestion would be very much appreciated.

**Comment by Hsilgos** [comment]
Hi, Hubert.
On GitHub we register only external issues. Internal one we collect in company's Jira.
You may check our activity through closed merge requests.

**Reply by Hsilgos** [reply]
I'll copy one of the pretty simple issues to GitHub

**Comment by Hsilgos** [comment]
Ok, I created new one. Thanks for attempt.

**Comment by hliberacki** [comment]
Hi Yauheni!

Thanks for such a quick reply and exporting internal issue to GitHub.
Now it clarifies why the issue backlog was short :)

Later I'll assign it to myself in order to understand the project by contribution.

Thanks!
